### PR TITLE
Add Support to retrieve HBAR/USD exchange Rate and volume from PayBito exchange

### LINKED ISSUE #141: Add PayBito HBAR/USD pair to ERT
**Summary**
PayBito has an HBAR/USD pair that should be added to the list of exchanges

https://trade.paybito.com/view-exchange/#/?pair=hbar-usd

Price & volume endpoint - https://trade.paybito.com/api/trades/HBAR_USD

### PR DESCRIPTION
**Pull request by anighanta**
**Related issue(s)**:
Closes #141 

**Summary of the change**:
Added a new class to read the data retrieved from the PayBito API.
Set the API in the config file that will be used in Mainnet,Testnet and PreProd networks.

**Applicable documentation**
- [x] Javadoc
- [x] tests
- [x] README

### CODE REVIEW

File: src/main/java/com/hedera/exchange/database/ExchangeRateAWSRD.java [hunk 1]
```diff
@@ -81,26 +81,6 @@
 
 	private static final String LATEST_ADDRESSBOOK_QUERY = "SELECT e1.expirationTime, e1.addressBook FROM address_book AS e1 INNER JOIN (SELECT MAX(expirationTime) expirationTime FROM address_book WHERE networkName = ?) AS e2 ON e1.expirationTime = e2.expirationTime LIMIT 1";
```
Comment (cesaragv) [inline src/main/java/com/hedera/exchange/database/ExchangeRateAWSRD.java:82]: Please open a ticket and assign it to me so I can check why the query fails if it is multiline

File: src/test/java/com/hedera/exchange/ERTprocTestCases.java [hunk 2]
```diff
@@ -269,18 +270,23 @@ public void testWeightedMedian() throws Exception {
         when(mockBinance.getHBarValue()).thenReturn(0.095);
         when(mockBinance.getVolume()).thenReturn(1000.0);
 
+        PayBito mockPayBito = mock(PayBito.class);
```
Comment (cesaragv) [inline src/test/java/com/hedera/exchange/ERTprocTestCases.java:273]: where do you assert these values?

File: src/test/java/com/hedera/exchange/ERTprocTestCases.java [hunk 3]
```diff
@@ -302,12 +308,17 @@ public void testWeightedMedian() throws Exception {
         when(mockBinance.getHBarValue()).thenReturn(0.035);
         when(mockBinance.getVolume()).thenReturn(1000.0);
 
+        PayBito mockPayBito = mock(PayBito.class);
```
Comment (cesaragv) [inline src/test/java/com/hedera/exchange/ERTprocTestCases.java:311]: where do you assert these values?

**Review by cesaragv — APPROVED** [review]
I would suggest changing the file Triage_ERT_Alerts.docx to be md, or another format which can be viewed online.

File: src/test/java/com/hedera/exchange/ERTprocTestCases.java (hunk 2 above)
Comment (anighanta) [reply src/test/java/com/hedera/exchange/ERTprocTestCases.java:273]: This value is used to calculate the median. So we assert the median instead to see if the value is consistent with this change.

File: src/test/java/com/hedera/exchange/ERTprocTestCases.java (hunk 3 above)
Comment (anighanta) [reply src/test/java/com/hedera/exchange/ERTprocTestCases.java:311]: This value is used to calculate the median. So we assert the median instead to see if the value is consistent with this change.

File: src/main/java/com/hedera/exchange/database/ExchangeRateAWSRD.java (hunk 1 above)
Comment (anighanta) [reply src/main/java/com/hedera/exchange/database/ExchangeRateAWSRD.java:82]: https://github.com/hashgraph/hedera-exchange-rate-tool/issues/132

**Comment by anighanta** [comment]
> I would suggest changing the file Triage_ERT_Alerts.docx to be md, or another format which can be viewed online.

But then I can't show images right ?

**Review by cesaragv — APPROVED** [review]
Maybe html would work with the images instead of docx

**Comment by nathanklick** [comment]
> Maybe html would work with the images instead of docx

Why not use markdown? Markdown supports images embedded just as easily as HTML.

**Comment by nathanklick** [comment]
Take a look at the PostgreSQL Guide in the Open Review repository for an example.

**Comment by anighanta** [comment]
@nathanklick @cesaragv  Does the new Triaging document look good?

**Review by cesaragv — APPROVED** [review]
Overall the document looks good, but most of the images look blurry. I would recommend opening a new issue to replace the images. 
Also, for some alerts, it would be nice to have more information. For instance, if an exchange rate was rejected, also show the value sent, i.e., in the case the alert was cause of a value we generated, add a message with the value sent, and the expected bounds.

File: docs/Triage_ERT_Alerts.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+## Identify the Network
+
+Alert specifies a network name in the prefix: MAINNET, TESTNET, PREPROD respectively for 
+Mainnet, TestNet and Pre-Production networks. 
+
+For example:
+This is an alert from Mainnet ERT saying that the Exchange Rate Update transaction sent to the Mainnet Nodes came back 
+with the status MAINNET_CALCULATED_MEDIAN_OUT_OF_BOUND.
+
+![](MAINNET_MedianOutOfBound.png)
```
[the remark this answers is not in the document]
Comment (anighanta) [reply docs/Triage_ERT_Alerts.md:14]: They look good when we look at the web version of it. I think if shrunk too much the image gets blurry

[the remark this answers is not in the document]
Comment (anighanta) [reply docs/Triage_ERT_Alerts.md:14]: Will crate an issue to make modifications to the document

**Pull request merged**